students/MikeShand/lesson04/file_lab_list.py

```diff
@@ -0,0 +1,4 @@
+import pathlib
+
+for file in pathlib.WindowsPath().rglob("*"):
+    print(file.absolute())

```

Comment:
1. don't use `file` as a var name, you can see github highlights it differently that is because it is a built-in function
2. how could you make it platform independant?